android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiUIWebView.java

```diff
@@ -931,4 +931,11 @@ protected void disableHWAcceleration()
 	{
 		Log.d(TAG, "Do not disable HW acceleration for WebView.", Log.DEBUG_MODE);
 	}
+
+	public void setMediaPlaybackRequiresUserGesture(boolean enabled)
+	{
+		if (Build.VERSION.SDK_INT >= 17) {
+			getWebView().getSettings().setMediaPlaybackRequiresUserGesture(enabled);
+		}
+	}
```

Comment:
Also, ideally, this should be a property (getter/setter) in JavaScript and not a method. Having a setter without a getter is not good design (but a getter-only API is okay).